proseco/core.py

```diff
@@ -561,6 +562,34 @@ def from_pickle(cls, obsid, rootdir='.'):
         with open(infile, 'rb') as fh:
             return pickle.load(fh)
 
+    def has_column_spoiler(self, cand, stars, mask=None):
+        """
+        Returns True if ``cand`` has a column spoiler downstream toward readout
+        register which is:
+        - at least 4.5 mag brighter (CHAR.col_spoiler_mag_diff)
+        - within 10 pixels in column (CHAR.col_spoiler_pix_sep)
+
+        Also records a log entry for detected spoilers
+
+        :param cand: object with 'row', 'col', and 'mag' items
+        :param stars: StarsTable
+        :param mask: boolean mask of stars to check (default=None => no mask)
+        :returns: bool
+        """
+        if mask is None:
+            mask = ()  # No mask
+
+        bads = ((np.abs(cand['row']) < abs(stars['row'][mask])) &  # Further from readout register
+                (np.sign(cand['row']) == np.sign(stars['row'][mask])) &  # Same side of CCD
```

Comment:
I suppose for acq and guide to be perfectly correct you'd also need to check both "sides" of the CCD in cases of candidate stars within 200 arsecs (or dither for guide) of the center of the CCD, right?  But I suppose even these column spoiler checks are somewhat overkill.

Reply:
Right, though the right number is more like 60 to 80 arcsec, which is the typical max maneuver error.  I think we can let this slide.